src/components/ui/Tag.js

```diff
@@ -0,0 +1,42 @@
+import styled from 'styled-components'
+
+const UnstyledTagList = ({ children, className }) => (
+  <ul className={className}>{children}</ul>
+)
+
+const TagList = styled(UnstyledTagList)`
+  padding: 0;
+  list-style: none;
+`
+
+const UnstyledTag = ({ children, color, backgroundColor, className }) => (
+  <li
+    style={{
+      color: color ?? 'default',
+      backgroundColor: backgroundColor ?? 'default',
+    }}
+    className={className}
+  >
+    {children}
+  </li>
+)
+
+const Tag = styled(UnstyledTag)`
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  box-sizing: border-box;
+  border-radius: 12px;
+  height: 23px;
+  padding: 0 10px;
+  font-size: 12px;
+  font-weight: bold;
+  background: ${({ theme }) => theme.transparentOrange};
+  color: ${({ theme }) => theme.orange};
+
+  &:not(:last-child) {
+    margin-right: 6px;
+  }
+`
```

Comment:
Same situation here. You can work off of `styled.li` directly, and change color and background color via `$(props) => props.color`, `$(props) => props.backgroundColor`

Reply:
So you wouldn't need UnstyledTag at all.